### ISSUE TITLE
[FEATURE] tickbase

### ISSUE THREAD

**Issue by larryngton** [CCBlueX/LiquidBounce#2746]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

please improve tickbase so it doesnt only work on vulcan and sentinel

**Comment by superblaubeere27** [comment]
Are there even clients that manage to bypass that?

**Comment by 1zun4** [comment]
Yes there are. In order to make tickbase work, it has to wait on pre-tickbase, unlike the current one which does it post.

**Comment by larryngton** [comment]
> Yes there are. In order to make tickbase work, it has to wait on pre-tickbase, unlike the current one which does it post.
> But the way Tickbase and TimerRange are implemented currently are simply bad and do not really give much benefit.

timerrange does work though, even if it needs some improvement
